frontend/reports/poverty-line/poverty-line-results.component.tsx

```diff
@@ -70,4 +72,41 @@ export default function PovertyLineResults(props) {
       ]}
     />
   );
+
+  function getCsvOptions(): Promise<CsvOptions> {
+    const allRow = {
+      Zip: "All zips and clients",
+      "Clients below Poverty Line": sum(
+        values(data.results.clientsBelowPovertyLine)
+      ).toLocaleString(),
+      "Total Clients": data.results.totalClients.toLocaleString(),
+      Percentage: formatPercentage(
+        sum(values(data.results.clientsBelowPovertyLine)),
+        data.results.totalClients
+      ),
+    };
+
+    return Promise.resolve({
+      columnNames: [
+        "Zip",
+        "Clients below Poverty Line",
+        "Total Clients",
+        "Percentage",
+      ],
+      data: Object.keys(data.results.clientsByZip)
+        .map((zip) => ({
+          Zip: zip,
+          "Clients below Poverty Line": (
+            data.results.clientsBelowPovertyLine[zip] || 0
+          ).toLocaleString(),
+          "Total Clients": data.results.clientsByZip[zip].toLocaleString(),
+          Percentage: formatPercentage(
+            data.results.clientsBelowPovertyLine[zip] || 0,
+            data.results.clientsByZip[zip]
+          ),
+        }))
+        .concat(allRow),
+      fileName: "Poverty_Line.csv",
```

Comment:
Let's change the name of the report to include the year of the poverty line that was used in the report. That can be found at `data.reportParameters.povertyLineYear`

```suggestion
      fileName: `Poverty_Line_${data.reportParameters.povertyLineYear}.csv`,
```